content/soarian-financials/stu3/structure-definition.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
I am not going to look at this file at this time, apply all comments left on patient.md here